Resources/ServerInfo/Guidebook/Antagonist/Revolutionaries.xml

```diff
@@ -40,7 +40,7 @@
   - [bold]Visibly be destroyed upon being implanted into a [color=#5e9cff]Head Revolutionary[/color][/bold], giving you away
   - NOT protect against flash disorientation
 
-  Assume all of [color=#cb0000]Security[/color] and [color=#1b67a5]Command[/color] are implanted with mindshields already, [bold]however they can be removed using an empty implanter, obtainable from the Medical department's MedFab.[/bold]
+  Assume all of [color=#cb0000]Security[/color] and [color=#1b67a5]Command[/color] are implanted with mindshields already, [bold]however - with the exception of the captains - they can be removed using an empty implanter, obtainable from the Medical department's MedFab.[/bold]
```

Comment:
```suggestion
  Assume all of [color=#cb0000]Security[/color] and [color=#1b67a5]Command[/color] are implanted with mindshields already. [bold]All mindshields other than the Captain's can be removed using an empty implanter.[/bold] You can make these with a medical techfab, which is usually inside the medical department.
```

Reply:
(Grammar note on how SS14 staff work: "John, a botanist, found work on a NT space station as a Botanist. When he was promoted to run the Service department, he became a Head of Personnel. He joined an inter-station workgroup on appropriate recycling of paper. All the other heads of personnel found his training as a botanist, his time as a Botanist and his current experience of being Head of Personnel invaluable.")